CONTRIBUTING.md

```diff
@@ -0,0 +1,177 @@
+# spacectl Contributing Guide
+
+In this guide you will get an overview of the contribution workflow from opening an issue, creating a pull request, to having it reviewed, merged and deployed.
+
+We welcome all contributions, no matter the size or complexity. Every contribution helps and is appreciated.
+
+The following is a set of guidelines for contributing to spacectl. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## Reporting an Issue
+
+If you spot a problem with spacectl [search if an issue already exists](https://github.com/spacelift-io/spacectl/issues). If a related issue doesn't exist, please open a new issue using [the issue form](https://github.com/spacelift-io/user-documentation/issues/new).
+
+## Making Changes
+
+spacectl is written in Go, and uses the [Spacelift GraphQL API](https://docs.spacelift.io/integrations/api) to interact with Spacelift. To make changes to spacectl you will need at least the following:
+
+- A working copy of [Go 1.22](https://go.dev/).
```

Comment:
We could just as to refer to the go.mod instead of pinning the version here.

```suggestion
- A working copy of [Go](https://go.dev/). Check `go.mod` to see what is the current version used by this project.
```

Just a nit, to save some maintenance effort.